nestkernel/nest_time.cpp

```diff
@@ -198,11 +197,11 @@ Time::reset_to_defaults()
 
 std::ostream& operator<<( std::ostream& strm, const Time& t )
 {
-  if ( t.tics == Time::LIM_NEG_INF.tics )
+  if ( t.tics <= Time::LIM_NEG_INF.tics )
```

Comment:
@jakobj I know this is getting a bit nit-picky, but since we are at it, I think we should beautify this to
```c++
if ( t.is_neg_inf() )
```
and the same below for the positive infinity case.